Hunting Queries/SecurityEvent/SignedBinaryProxyExecutionRundll32.yaml

```diff
@@ -0,0 +1,41 @@
+id: c2074fce-b5ba-4c0a-9332-d08b8fc43c53
+name: Rundll32 (LOLBins and LOLScripts)
+description: |
+  'This detection uses Sysmon telemetry to hunt Signed Binary Proxy Execution: Rundll32 activities'
+
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - Defense Evasion
+relevantTechniques:
+  - T1218.011
+
+query: |
+  Event
+  //This query uses sysmon data depending on table name used this may need updataing
+  | where Source == "Microsoft-Windows-Sysmon"
+  | where EventID == 1
+  | extend RenderedDescription = tostring(split(RenderedDescription, ":")[0])
+  | extend EventData = parse_xml(EventData).DataItem.EventData.Data
+  | mv-expand bagexpansion=array EventData
+  | evaluate bag_unpack(EventData)
+  | extend Key=tostring(['@Name']), Value=['#text']
```

Comment:
This query will fail execution and rule creation if we don't have a recent event or an event within the timeframe.  Changing this line to - `| extend Key = tostring(column_ifexists('@Name', "")), Value = column_ifexists('#text', "")` - will allow it to not fail.

Reply:
Done. Thank you @shainw